share/completions/mtr.fish

```diff
@@ -0,0 +1,41 @@
+complete -c mtr -x -a "(__fish_print_hostnames)"
+
+complete -c mtr -s h -l help -d 'display this help and exit'
+complete -c mtr -s v -l version -d 'output version information and exit'
+complete -c mtr -s 4 -d 'use IPv4 only'
+complete -c mtr -s 6 -d 'use IPv6 only'
+complete -c mtr -s F -l filename -d 'read hostname(s) from a file' -rF
+complete -c mtr -s r -l report -d 'output using report mode'
+complete -c mtr -s w -l report-wide -d 'output wide report'
```

Comment:
These should be explained. From [the man page](https://github.com/traviscross/mtr/blob/master/man/mtr.8.in), it seems like "wide report" mode means that the hostnames won't be abbreviated?

So something like "Output a statistic report" for `--report` and "Output report with full hostnames" for `--report-wide`?